cfme/configure/settings.py

```diff
@@ -53,272 +50,339 @@ def __init__(self, description=None, scope=None, days=None, hours=None, timezone
         self.timezone = timezone
 
     def create(self, cancel=False):
-        navigate_to(self, 'Add')
-        fill(self.timeprofile_form, {'description': self.description,
-                                     'scope': self.scope,
-                                     'days': self.days,
-                                     'hours': self.hours,
-                                     'timezone': self.timezone,
-                                     })
+        view = navigate_to(self, 'Add')
+        new_timeprofile = view.timeprofile_form.fill({
+            'description': self.description,
+            'scope': self.scope,
+            'days': self.days,
+            'hours': self.hours,
+            'timezone': self.timezone,
+        })
+
         if not cancel:
-            sel.click(self.save_button)
+            view.timeprofile_form.save_button.click()
             end = "saved" if self.appliance.version > '5.7' else "added"
-            flash.assert_success_message('Time Profile "{}" was {}'
-                                         .format(self.description, end))
+            FlashMessages('Time Profile "{}" was {}'.format(self.description, end))
+        return new_timeprofile
```

Comment:
This method returns bool, is it required?

Reply:
The test case which are based on this method uses that return value, hence I have included this return statement.
The line which uses this: https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/configure/test_timeprofile.py#L29

Reply:
The line which you provided is using for `copy()` method, but here is `create()` method.

Reply:
Sorry I messed up with the line nos. Will make the changes as said.